tests/sdk/synthesizers/test_pytorch_synthesizer.py

```diff
@@ -53,4 +53,18 @@ def test_sample(self):
         self.dpctgan.fit(df, categorical_columns=['sex','educ','race','married'])
         sample_size = len(df)
         synth_data = self.dpctgan.sample(sample_size)
+        assert synth_data.shape == df.shape
+
+class TestPytorchDPSynthesizer_PATECTGAN:
+    def setup(self):
+        self.patectgan = PytorchDPSynthesizer(PATECTGAN(), None)
```

Comment:
Given the change in name to opacus, this is a reasonable name now :) Which is niced. The is Pytorch DP  with PATE. No package name conflict anymore after we update to opacus